gtfs/spec/en/reference.md

```diff
@@ -233,7 +235,7 @@ File: **Conditionally Required**
 
 |  Field Name | Type | Presence | Description |
```

Comment:
Should this set of modifications also clarify that "the end_date date must not precede the start_date date"? Or would this happen in a separate PR?

Reply:
Noted for a separate PR.